switchlink/switchlink.h

```diff
@@ -23,7 +23,7 @@
 
 #include "switchutils/switch_utils.h"
 
-#define switchlink_malloc(x, c) malloc(x *c)
+#define switchlink_malloc(x, c) malloc((x) * (c))
```

Comment:
Parenthesize macro parameters to guard against operator precedence errors if either of the actual parameter values is an expression.